### PR TITLE
#438 Szusivacsi new show

### PR DESCRIPTION
**Pull request by baxgas**

### CODE REVIEW

File: azuracast/liquidsoap_custom.liq [hunk 1]
```diff
@@ -383,6 +386,7 @@ radio = switch(track_sensitive=false, transitions=[
     transition_into_show(playlist_jingle_station_id),
     transition_into_show(playlist_jingle_station_id),
     transition_into_show(playlist_jingle_station_id),
+    transition_into_show(playlist_jingle_station_id),
```
Comment (gammaw) [inline azuracast/liquidsoap_custom.liq:389]: @baxgas I think this new line should be in the Friday block.

**Review by gammaw — CHANGES_REQUESTED** [review]
Thanks Gás. There seems to be one issue with this PR, otherwise it looks fine. Can you give it a look?

File: azuracast/liquidsoap_custom.liq (hunk 1 above)
Comment (baxgas) [reply azuracast/liquidsoap_custom.liq:389]: @gammaw – the Friday section begins on line 379 and ends on 390 as I see from the comments! 
`379    transition_into_off(playlist_jingle_station_id), #ambient: ^^^ Friday ^^^ 
`
`....`
`390    transition_into_off(playlist_jingle_station_id), #ambient: ^^^ Saturday ^^^
`

Comment (baxgas) [reply azuracast/liquidsoap_custom.liq:389]: @gammaw – oh, it is not correct, you were right, it has the comments at the END of the days blocks, I'll fix the position

**Review by gammaw — APPROVED** [review]
Thanks!

**Pull request merged**